packages/babel-preset-env/test/normalize-options.spec.js

```diff
@@ -23,6 +21,11 @@ describe("normalize-options", () => {
       ]);
     });
 
+    it("should not normalize babel-plugin with prefix", () => {
+      const normalized = normalizePluginName("prefix-babel-plugin-postfix");
```

Comment:
Imho this is kinda testing an implementation detail, could you rewrite it to test an actual effect rather than a single normalization step?

Reply:
It's a little bit tricky to write a test for the `include`/`exclude` field, because as far as I know there is no `prefix-babel-plugin-*`, therefore any test will end up throwing an error (invalid plugin) regardless of this behavior.

Reply:
makes sense, u'd have to mock something or create fake `node_modules` in test dir, not worth it then at this point